queries/elm/folds.scm

```diff
@@ -0,0 +1,13 @@
+((function_call_expr) @_fn
+  (#not-has-parent? @_fn parenthesized_expr)) @fold
+
+[
+  (case_of_branch)
+  (case_of_expr)
+  (value_declaration)
+  (type_declaration)
+  (type_alias_declaration)
+  (list_expr)
+  (record_expr)
+  (parenthesized_expr)
```

Comment:
Can we please also get `(import_clause)+`?

Reply:
Added